app/models/solidus_paypal_braintree/customer.rb

```diff
@@ -1,4 +1,7 @@
 class SolidusPaypalBraintree::Customer < ApplicationRecord
   belongs_to :user, class_name: Spree::UserClassHandle.new
   has_many :sources, class_name: "SolidusPaypalBraintree::Source", inverse_of: :customer
+
+  validates :user_id, presence: true
+  validates :braintree_customer_id, presence: true
```

Comment:
What about existing records? Could we end up saving records previously created that now are no more valid and can't be updated anymore?

Reply:
Yeah, this has been removed